data/data/aws/main.tf

```diff
@@ -77,3 +77,12 @@ module "vpc" {
 
   tags = "${local.tags}"
 }
+
+resource "aws_ami_copy" "main" {
```

Comment:
Is there enough information for users to see that this AMI was created from a openshift trusted AMI?

Reply:
> Is there enough information for users to see that this AMI was created from a openshift trusted AMI?

To see where?  In the AWS console?

Reply:
console, get ami from cli; somewhere that a user can look.

Reply:
> console, get ami from cli; somewhere that a user can look.

The source AMI ID does not seem to be listed anywhere, although we could add a tag for that if we wanted (just let me know).  Both the console and CLI show the description inherited from the source, e.g.:

```console
$ aws ec2 describe-images --image-ids ami-02600559c19681dd4 --query 'Images[].Description' --output text
Red Hat CoreOS "maipo" (OpenShift) 47.297
```

Reply:
Yeah, add a tag that provides the ami I'd used to generate this.

Reply:
> Yeah, add a tag that provides the ami id used to generate this.

Done (and I added a tag for the region too) with f08e266 -> 4fcef5e.